File: Easy/palindrome_number/palindrome_number.py
def isPalindrome(x):
    if x<0:
        return False
    
    result = 0
    #will be doing operations on num and keep original copy value the same
    num = x

    while num>0:
        remainder = num%10
        result = result*10 + remainder
        num = num//10
    
    return x==result
    

File: Easy/palindrome_number/palindrome_number.c
bool isPalindrome(int x){
    long remainder,result;
    if (x<0){
        return false;
    }
    int num = x;
    result = 0;
    while (x>0){
        remainder = x%10;
        result = result *10 + remainder;
            x /= 10;
    }
    return num==result;
}